newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -636,6 +637,7 @@ export default class AuthenticatedUserProvider extends React.Component<
     }
     this._markAuthenticatedUserAsLoggedOut();
     clearCloudProjectCookies();
+    burstCloudProjectAutoSaveCache();
```

Comment:
can you add this in the doDeleteAccount() function?
(maybe worth creating one single function onLogout() or something similar

Reply:
Good idea! I missed that